src/main/java/uk/gov/justice/digital/delius/service/AttendanceService.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.justice.digital.delius.service;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.justice.digital.delius.data.api.Attendance;
+import uk.gov.justice.digital.delius.data.api.Attendance.ContactTypeDetail;
+import uk.gov.justice.digital.delius.jpa.standard.entity.Contact;
+import uk.gov.justice.digital.delius.jpa.standard.repository.ContactRepository;
+
+@Slf4j
```

Comment:
looks like the log is never used, so could be removed if you are getting a compile warning

Reply:
Done